st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -474,7 +477,7 @@ def _resolve_params(action_node, original_parameters, results, chain_vars, chain
         context.update({SYSTEM_KV_PREFIX: KeyValueLookup()})
         context.update({ACTION_CONTEXT_KV_PREFIX: chain_context})
         try:
-            rendered_params = jinja_utils.render_values(mapping=action_node.params,
+            rendered_params = jinja_utils.render_values(mapping=action_node.get_parameters(),
```

Comment:
Here is an alternative approach using property getters - b8ee345af1b3243af4151d4ec9f9710cd4504b58

Advantage of this approach is that it allows us to consistently access `Node` class attributes. With the previous approach, we need to use getter for parameters (`node.get_parameters()`) and directly access instance variable for others (`node.name`).

So, dunno. What do others think?